doc/manuals/user/walkthrough_apiv1.md

```diff
@@ -988,17 +988,16 @@ EOF
 
 Let's examine in detail the different elements included in the payload:
 
--   entityIdList and attributeList ('entities' and 'attributes' for
-    short, in JSON) define which context elements will be included in
-    the notification message. 
+-   entities and attributes for short, in JSON) define which context 
```

Comment:
"for short, in JSON)" seems to be a leftover. I understand it should be removed.

Please, have a look to "for short" in the whole documentation, in order to locate similar cases that need to be fixed.